docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx

```diff
@@ -12,9 +12,5 @@ export default {
   component: Button,
 };
 
-export const Large = {
-  parameters: {
-    controls: { hideNoControlsWarning: true },
-  },
-};
-```
\ No newline at end of file
+export const Large = {};
```

Comment:
The file seems to suggest that it's all about hiding the controls warning. Is this file still needed or perhaps we need to change the file name?